.github/workflows/benchmarks.yml

```diff
@@ -133,10 +134,6 @@ jobs:
           admin_console_url=$(terraform output -raw admin_console_url)
           echo "admin_console_url=$admin_console_url" >> "$GITHUB_OUTPUT"
           echo "-> infra setup done"
-        env:
-          TF_VAR_worker_region: ${{ env.AWS_REGION }}
```

Comment:
`env.AWS_REGION` doesn't exist. Hence I removed it.

Reply:
can you add the link to the reference that included it? In case there are some rationale?

Reply:
https://github.com/elastic/apm-server/pull/13884/files#diff-22eef62c9d7b36d02f5fd5aaada92702e22d6795c464239cb7b8fe0f26ea1e1cR137

Reply:
is this what you meant?